Rules/UseConsistentIndentation.cs

```diff
@@ -146,10 +148,30 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                 switch (token.Kind)
                 {
                     case TokenKind.AtCurly:
-                    case TokenKind.AtParen:
-                    case TokenKind.LParen:
                     case TokenKind.LCurly:
+                        AddViolation(token, indentationLevel++, diagnosticRecords, ref onNewLine);
+                        break;
+
                     case TokenKind.DollarParen:
+                    case TokenKind.AtParen:
+                        lParenSkippedIndentation.Push(false);
+                        AddViolation(token, indentationLevel++, diagnosticRecords, ref onNewLine);
+                        break;
+
+                    case TokenKind.LParen:
+                        // When a line start with a parenthesis and it is not the last non-comment token of that line,
+                        // then indentation does not need to be increased.
+                        if ((tokenIndex == 0 || tokens[tokenIndex - 1].Kind == TokenKind.NewLine) &&
+                            (tokenIndex < tokens.Length - 1 && NextTokenIgnoringComments(tokens, tokenIndex)?.Kind != TokenKind.NewLine))
+                        {
+                            if (onNewLine)
+                            {
+                                onNewLine = false;
+                            }
```

Comment:
This logic means that onNewline will always be `false` on line 171 no? In that case, why not just put `onNewLine = false`?

Reply:
addressed in 5f2201e